spec/factories/conference_rooms.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :conference_room do
+    capacity 1
+    title "MyString"
```

Comment:
use sequence here due to uniqueness validation on those fields:
`sequence :title {|n| "Conf Room ##{n}" `
`sequence :color {|n| "#fff00#{n}" `